lib/bundler/definition.rb

```diff
@@ -409,8 +409,10 @@ def ensure_equivalent_gemfile_and_lockfile(explicit_flag = false)
              "updated #{Bundler.default_lockfile.relative_path_from(SharedHelpers.pwd)} to version control."
 
       unless explicit_flag
+
+        suggest_command = Bundler.settings.locations("frozen")[:global] == "1" ? "bundle config --delete frozen" : "bundle install --no-deployment"
```

Comment:
That's one other thing that I saw. I need to re-verify this but it looks like `bundle config --delete frozen` deletes BOTH local and global config.